packages/canvas-tokens/utils/transformers/index.ts

```diff
@@ -9,6 +9,14 @@ import {transformNameToCamelCase} from './transformNameToCamelCase';
 import {transformHexToRgb} from './transformHexToRgb';
 
 export const transforms: Record<string, Transform> = {
+  'value/deprecated': {
+    type: 'value',
+    transitive: true,
+    matcher: filter.isDeprecated,
+    transformer: ({value, fallback}) => {
+      return fallback || value;
```

Comment:
ahh ok so this should catch the fallback value is there's no value?

Reply:
it uses fallback first, but there is still no check for no values, no fallback